Pharma - Change KAM Painkillers to CWMP

Apricot-ale opened the pull request:
**When merged this pull request will:**
- _Fix https://github.com/KAT-Advanced-Medical/KAM/issues/546 problem._
- _Rename whole of (KAT) Painkillers thing (except item class name) to CWMP (from [TCCC 14.3](https://tccc.org.ua/en/guide/combat-wound-medication-pill-pack)) and item stringtable to Combat-Pill-Pack (from  [DD 1380 TCCC CASUALTY CARD](https://learning-media.allogy.com/api/v1/pdf/eb6c36b5-edbf-4e68-ae47-7a6e6536c6e7/contents))_
- _Max "ammo" count is now 10->4, weight is 1->0.5 (Guide need to change)_

New icon;
![icon_painkillers](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/41a0e77c-ce68-4724-8526-ec888f114802)
New icon with higher resolutions (you know where it come from...)
when 64x64, almost text are can't read and texture not detailed so it's will be ok
![icon_painkillers_256](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/81af1dba-0d95-4ed9-a42c-a687f5e8f2c7)



### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

Apricot-ale committed:
Painkiller Treatment renaming

Rename Painkillers Treatment as KAT_Painkillers for avoid conflict with ACE Painkiller

MiszczuZPolski commented on addons/pharma/XEH_PREP.hpp:
```diff
@@ -36,7 +36,7 @@ PREP(treatmentAdvanced_MorphineLocal);
 PREP(treatmentAdvanced_Naloxone);
 PREP(treatmentAdvanced_NaloxoneLocal);
-PREP(treatmentAdvanced_Painkillers);
+PREP(treatmentAdvanced_KAT_Painkillers);
```
There is no need to add KAT here

Apricot-ale committed:
revert the treatmentAdvanced name change

Apricot-ale committed:
*THIS COMMIT IS TEST* TriageCard check

THIS COMMIT IS USED AS TEST PURPOSE. THIS WILL BE REVERT AFTER CHECK/DEBUG PLAYING.

Apricot-ale committed:
Revert "*THIS COMMIT IS TEST* TriageCard check"

This reverts commit e1d2645cea52f0b3bd752b325a40cbd6fa4968e1.

MiszczuZPolski committed:
change tylenol to CWMP from TCCC guidelines

MiszczuZPolski committed:
Update XEH_PREP.hpp

Apricot-ale committed:
change painkiller name to CWMP

just showcase. might be revert

Apricot-ale committed:
Change wording "Combat Pill Pack"

Apricot-ale committed:
add Japanese...

Apricot-ale committed:
add "pill" to descshort

Apricot-ale committed:
Change item icon

Apricot-ale committed:
change suppress pain wording

Painkiller now suppresses pain

MiszczuZPolski committed:
Merge branch 'dev-Tomcat' into PainkillerRename-Jun24-01

MiszczuZPolski committed:
remove line
